rcon/steam_utils.py

```diff
@@ -15,6 +17,39 @@
 logger = logging.getLogger(__name__)
 
 last_steam_api_key_warning = datetime.datetime.now()
+steam_id_64_pattern = re.compile(r"\d{17}")
+
+
+def is_steam_id_64(steam_id_64: str) -> bool:
```

Comment:
Would probably be more performant if we do not use regular expressions for that. A Steam ID right now is always only digits and always 17 characters long. Something like this should be alright:

```
if steam_id_64.isdigit() and len(steam_id_64) == 17:
    return True
return False
```

Reply:
Much smarter